File: machi_koro/cards/secondary_industry.py
import machi_koro.cards.establishment_base as establishment_base
import machi_koro.utils as utils
import machi_koro.card_effects.establishments.secondary as card_effect_secondary

from dataclasses import dataclass


@dataclass
class SecondaryIndustryCard(establishment_base.EstablishmentBase):
    card_type: utils.CardType = utils.CardType.SecondaryIndustry
    card_color: utils.CardColor = utils.CardColor.GREEN
    card_effect: card_effect_secondary = card_effect_secondary.CardEffectSeconday

File: MachiKoro.Core/Models/Cards/Establishments/Basic/SecondaryIndustry.cs
using MachiKoro.Core.CardEffects.Establishments;
using MachiKoro.Core.CardEffects.Establishments.Basic;
using MachiKoro.Utilities;
using System.Collections.Generic;

namespace MachiKoro.Core.Cards.Establishments.Basic
{
    public class SecondaryIndustryCard : EstablishmentBase
    {
        private readonly SecondayCardEffect _secondayCardEffect;

        public SecondaryIndustryCard(string name, CardCategory cardIcon, List<int> activationNumbers, int constructionCost, SecondayCardEffect cardEffectSeconday) 
            : base (name, cardIcon, activationNumbers, constructionCost, cardEffectSeconday)
        {
            _secondayCardEffect = cardEffectSeconday;
        }
    }
}
